helpdesk_mgmt_sale/models/helpdesk_ticket.py

```diff
@@ -17,9 +17,9 @@ def _compute_so_count(self):
     def action_view_sale_orders(self):
         """Returns action to view sale orders related to this ticket."""
         action = self.env["ir.actions.actions"]._for_xml_id("sale.action_orders")
-        action["domain"] = [("ticket_id", "=", self.id)]
+        action["domain"] = [("ticket_ids", "in", [self.id])]
         action["context"] = {
-            "default_ticket_id": self.id,
+            "default_ticket_ids": [(6, 0, [self.id])],
```

Comment:
I think this won't work

Reply:
why not?
![ticket](https://github.com/user-attachments/assets/c633221f-8e17-43ef-ab16-8af30ff298f3)

Reply:
For not losing existing data:

```suggestion
            "default_ticket_ids": [(4, self.id)],
```

Reply:
Actually with the tests I did, although the context is transmitted to the view it is only affecting when creating a new order and not the existing ones but you are right, the right way is to make sure that the existing ones are not overwritten. The change is done, thanks